src/sidebar/util/test/oauth-client-test.js

```diff
@@ -162,15 +162,35 @@ describe('sidebar.util.oauth-client', () => {
     });
   });
 
+  describe('.createLoginPopupWindow', () => {
+    it('creates and returns the popup window', () => {
+      var fakeWindow = new FakeWindow;
+      var popupWindow = OAuthClient.createLoginPopupWindow(fakeWindow);
+      assert.equal(popupWindow, fakeWindow.open.returnValues[0]);
+      assert.calledWith(
+        fakeWindow.open,
+        'about:blank',
+        'Login to Hypothesis',
+        'height=430,left=274.5,top=169,width=475'
+      );
+    });
+  });
+
   describe('#authorize', () => {
     var fakeWindow;
 
     beforeEach(() => {
       fakeWindow = new FakeWindow;
     });
 
-    it('opens a popup window at the authorization URL', () => {
-      var authorized = client.authorize(fakeWindow);
+    function authorize() {
+      var popupWindow = OAuthClient.createLoginPopupWindow(fakeWindow);
+      var authorized = client.authorize(fakeWindow, popupWindow);
```

Comment:
There's a coupling between `createLoginPopupWindow` and `authorize` — `authorize` now requires the second param (`authWindow`) to be successful. Is there any value in testing for `authorize`'s behavior if the second (popup window) argument is omitted or invalid in some way?